include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -522,8 +525,9 @@ SpatialOperatorBase<dim, Number>::initialize_operators(std::string const & dof_i
 
   data.bc = boundary_descriptor->velocity;
 
-  data.dof_index  = get_dof_index_velocity();
-  data.quad_index = get_quad_index_velocity_linearized();
+  data.dof_index           = get_dof_index_velocity();
+  data.quad_index          = get_quad_index_velocity_linearized();
+  data.quad_index_standard = get_quad_index_velocity_linear();
```

Comment:
Shouldn't this be a parameter of `ViscousKernelData`? As a positive side-effect, the interface of `ViscousKernel::reinit()` would not need to be changed and hopefully also `MomentumOperatorData` does not have to be touched here. It's a long time ago, so maybe I overlook something right now ...